content/en/docs/guides/image-update.md

```diff
@@ -627,396 +627,99 @@ flux create image policy podinfo \
-metadata:
-  name: gcr-credentials-sync
-  namespace: flux-system
-subjects:
-- kind: ServiceAccount
-  name: gcr-credentials-sync
-roleRef:
-  kind: Role
-  name: gcr-credentials-sync
-  apiGroup: ""
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  annotations:
-    iam.gke.io/gcp-service-account: <name-of-service-account>@<project-id>.iam.gserviceaccount.com
-  name: gcr-credentials-sync
-  namespace: flux-system
----
-apiVersion: batch/v1beta1
-kind: CronJob
-metadata:
-  name: gcr-credentials-sync
-  namespace: flux-system
-spec:
-  suspend: false
-  schedule: "*/45 * * * *"
-  failedJobsHistoryLimit: 1
-  successfulJobsHistoryLimit: 1
-  jobTemplate:
-    spec:
-      template:
-        spec:
-          serviceAccountName: gcr-credentials-sync
-          restartPolicy: Never
-          containers:
-          - image: google/cloud-sdk
-            name: create-secret
-            imagePullPolicy: IfNotPresent
-            env:
-            - name: SECRET_NAME
-              value: gcr-credentials
-            - name: GCR_REGISTRY
-              value: <REGISTRY_NAME> # fill in the registry name e.g gcr.io, eu.gcr.io
-            command:
-            - /bin/bash
-            - -ce
-            - |-
-              kubectl create secret docker-registry $SECRET_NAME \
-                --dry-run=client \
-                --docker-server="$GCR_REGISTRY" \
-                --docker-username=oauth2accesstoken \
-                --docker-password="$(gcloud auth print-access-token)" \
-                -o yaml | kubectl apply -f -
-```
-
-Since the cronjob will not create a job right away, after applying the manifest,
-you can manually create an init job using the following command:
-
-```sh
-$ kubectl create job --from=cronjob/gcr-credentials-sync -n flux-system gcr-credentials-sync-init
-```
-
-After the job runs, a secret named `gcr-credentials` should be created. Use this
-name in your GCR ImageRepository resource manifest as the value for
-`.spec.secretRef.name`.
-
+patches:
+- target:
+    version: v1
+    group: apps
+    kind: Deployment
+    name: image-reflector-controller
+    namespace: flux-system
+  patch: |-
+    - op: add
+     path: /spec/template/spec/containers/0/args/-
+     value: --aws-autologin-for-ecr
+```
+
+#### Using Native GCP GCR Auto-Login
+
+There is native support for the GCP Google Container Registry available since `image-reflector-controller` [vadd-version](link to changelog)
+which was released with Flux release v[add-flux-ver]. This feature is enabled by setting the `--gcp-autologin-for-gcr`
+flag. This works with both clusters that have Workload Identity enabled, and those that use the default service account.
+This flag can be added by including a patch in the `kustomization.yaml` overlay file in your `flux-system`,
+similar to the process described in [customize Flux manifests](../installation.md/#customize-flux-manifests):
+                                             
```

Comment:
This would be a good place to mention/add the "this guide" link I mentioned before.